### DISCUSSION TITLE
discordで挙がっていた「死亡後復活してもスコアを送信できるようにする」オプションについての話題

### DISCUSSION THREAD

**Discussion by chikatei** [hengband/hengband#787] [General]
初心者向けのチュートリアルや新規プレイヤーへのハードルを下げる措置の一環として、死亡後に復活したデータでもスコアを送信できるようにしてはどうか、という話題がdiscordで挙がっていました。

このまま流れてしまうと勿体ないと思ったので、ログの一部を転載させていただいて記録しておこうと思います。
（元の発言と文意は変わらないようにしていますが、一部削ったり補足したりしています）

●死亡時に復活できるようにする、死亡時のペナルティを緩和（現状の死を回避するオプションと同等？）
・死んでも辺境に戻されるだけ
・スコア送信禁止と引き換えに復活
・死んだときに「復活しますか？」と確認

●死亡時に復活するしスコアの送信も可能にする
・dumpに死の記録が残る
・死亡回数が記録される
・（死亡後復活しても）スコアサーバーに載っても良い
・ぱっと見で復活したことが分かるようにクローンナンバーを付ける
・○度復活したPLAYER
・1回目の死亡時のみスコア送信可能
・1回目死亡時にスコア送れて、次に送れるのはクリア時
・スコアで競うのは死んでない人のみで、あとはスコアを送ったら死亡回数順に並ぶ
・ノーコンティニューの部とcontinue回数の部でスコアを切り分ける感じで
・死んだらスコア0

**Comment by chikatei** [comment]
これはログの転載とは別に私の個人的な意見ですが、
現状でも
「死を回避することを可能にする（詐欺オプション）」、
「初心者用簡単モード（初期オプション）」、
「いかさま（性格）」など
ゲームの難易度を下げるための設定は既に豊富なので、
それらの存在を初心者に分かりやすい位置に置くとか、
心情的に選びやすい名前に改名するとか、
初心者救済としてはそういった方法も考えられるかもしれませんね。

**Comment by deskull-m** [comment]
初心者がとっつきやすくするという方針そのものは、私も大概重んじたいところなのですが変愚でどこまでやるかは考える所ではありますね。仰ったところ含めて何かは考えたいと思います。
